### PR TITLE
refactor: Re-arranging folder structure

### PR DESCRIPTION
**Pull request by rajsam003**
Co-authored-by: Angela Patrinou <36475005+patrinoua@users.noreply.github.com>
Co-authored-by: Pavlos Vinieratos <pvinis@gmail.com>

### PR Checklist

- [x] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [x] I added Tests and Stories for my changes.

### To the reviewers 👀

- [x] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

### CODE REVIEW

**Comment by patrinoua** [comment]
Why is my co-authored email weird? 🤔

**Comment by pvinis** [comment]
@patrinoua thats the email i see on your commits too. probably somewhere in the gh settings? or maybe in the git config on your machine?

**Review by pvinis — APPROVED** [review]
nice. merge when you want.

File: alias.js [hunk 1]
```diff
@@ -1,15 +1,14 @@
 const list = [
+  "app/screens",
```
Comment (pvinis) [inline alias.js:2]: i think this can just be `"app"`, and nothing more.

**Pull request merged**